samplingsimulatorpy/generate_virtual_pop.py

```diff
@@ -34,19 +34,22 @@ def generate_virtual_pop(size, distribution_func, *para):
     >>> from samplingsimulatorpy import generate_virtual_pop
     >>> pop = generate_virtual_pop(100, np.random.normal, 0, 1)
     """
-    
-    if (size <= 0) :
+
+    if (size <= 0):
         raise ValueError("Size of population must be a positive integer")
 
-    if (not isinstance(size, int)) :
+    if (not isinstance(size, int)):
         raise TypeError("Size of population must be a positive integer")
 
     try:
         distribution_func(*para, size)
     except TypeError:
-        print('Please enter a valid distribution function with correct number of parameters for the distribution function')
+        print('Please enter a valid distribution function'
+              'with correct number of parameters for'
+              'the distribution function')
         raise TypeError
 
-    pop = pd.DataFrame(distribution_func(*para, size=size), columns=[distribution_func.__name__])
+    pop = pd.DataFrame(distribution_func(*para, size=size),
+                       columns=[distribution_func.__name__])
```

Comment:
If you add `var_name` as an argument, then this line could change the name of the column to that name.

```suggestion
    pop = pd.DataFrame(distribution_func(*para, size=size),
                       columns=[var_name)
```

Reply:
I will update the full version

Reply:
Fair enough